frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -179,6 +182,14 @@ export const AccountsSummary = ({
             <Status hidden={!hasCard} type="warning">
               {t('warning.sdcard')}
             </Status>
+            {accountsByKeystore.map(({ keystore }) => (
+              <BackupReminder
+                key={keystore.rootFingerprint}
+                keystore={keystore}
+                deviceID={Object.keys(devices)[0]}
```

Comment:
I worked on the assumption that there is only one value here when the keystore is connected, as we can't have multiple BB02 connected at the same time. It works, but are there possibly cases I missed here?